### ISSUE TITLE
copy instruction is ignored

### ISSUE THREAD

**Issue by taukakao** [Vanilla-OS/Vib#65]
When using the following recipe:
```yml
name: Custom
id: custom

stages:
- id: build
  base: ghcr.io/vanilla-os/desktop:main
  modules:

  - name: example-commands
    type: shell
    copy:
    - paths:
      src: hello
      dst: /wow
    commands:
    - echo Example output
```

it builds successfully but the copy instruction is ignored. 
Might be fixed by: #63

**Comment by kbdharun** [comment]
This won't be fixed by #63, since both adds and copy directives currently doesn't have support to be defined inside a module, it must be defined in the top portion of stages before modules.

**Comment by taukakao** [comment]
You're right, my bad.
